custom_components/kia_uvo/sensor.py

```diff
@@ -78,6 +78,20 @@
         native_unit_of_measurement=PERCENTAGE,
         device_class=SensorDeviceClass.BATTERY,
     ),
+    SensorEntityDescription(
+        key="ev_battery_soh_percentage",
```

Comment:
I don't see these key values in https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/blob/master/hyundai_kia_connect_api/Vehicle.py

Without that this will fail. First it needs to be added to the vehicle, mapped in the region to that attribute.  Once done can be added here.

Reply:
Now the library should be fixed. I double checked :-)

Reply:
One note so you are aware.  Now that the API is updated the manifest.json file in this repo needs the new API version.  This PR should reflect that since it is required for the sensors to function.

Reply:
I changed the manifest to 3.14.1.

Reply:
Should I update the documentation? I think we might need a new column to show what is supported by the new EU version.

Reply:
Never a bad thing.  We always are a bit behind on the docs!

Reply:
I updated the readme. Let me know what you think.